autoload/codefmt/clangformat.vim

```diff
@@ -19,8 +19,16 @@ let s:plugin = maktaba#plugin#Get('codefmt')
 function! s:ClangFormatHasAtLeastVersion(minimum_version) abort
   if !exists('s:clang_format_version')
     let l:executable = s:plugin.Flag('clang_format_executable')
-    if codefmt#ShouldPerformIsAvailableChecks() && !executable(l:executable)
-      return 0
+    if codefmt#ShouldPerformIsAvailableChecks()
+      if !executable(l:executable)
+        return 0
+      elseif !empty(l:executable) && l:executable isnot# 'clang-format'
+        " The user has specified a custom formatter command. Hope it works.
+        " /shrug.
+        return 0
```

Comment:
This (and the branch immediately below) aren't correct: in both cases (custom executable and 'clang-format' being executable), we want to continue to call --version to check the reported version number.

The only case we want to short-circuit is the non-custom-executable case where the executable can't be run.

Reply:
I've updated the logic